api/src/orgs/resolver.js

```diff
@@ -1,4 +1,11 @@
-import { getOrganisations } from './data'
+import { getOrganisations, getSingleOrganisation, updateProfile } from './data'
+
+const EMPTY_PROFILE = {
```

Comment:
It seems like defaults are already set in the `Profile` resolver a bit further down, so there is no need for `EMPTY_PROFILE`

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/6280fdcc5b5f13aae91b538428c8ec1d83e4ccc4.